src/transaction/TransactionController.ts

```diff
@@ -749,23 +712,26 @@ export class TransactionController extends BaseController<TransactionConfig, Tra
       return;
     }
 
-    const [etherscanResponse, alethioResponse] = await handleTransactionFetch(networkType, address, opt);
+    const [etherscanTxResponse, etherscanTokenResponse] = await handleTransactionFetch(networkType, address, opt);
     const remoteTxList: { [key: string]: number } = {};
     const remoteTxs: TransactionMeta[] = [];
 
-    etherscanResponse.result.forEach((tx: EtherscanTransactionMeta) => {
-      /* istanbul ignore next */
+    /* istanbul ignore next */
+    etherscanTxResponse?.result.forEach((tx: EtherscanTransactionMeta) => {
```

Comment:
It looks like `etherscanTxResponse` _can't_ be `undefined` here, which probably explains the lack in coverage that prompted the `istanbul ignore next` comment.

Could you remove the `?` operator here instead of ignoring the coverage gap?

The same goes for the `etherscanTokenResponse` below.